### DISCUSSION TITLE
How to remove Online previews ?

### DISCUSSION THREAD

**Discussion by Vblgh-Bruno** [microsoft-search/pnp-modern-search#1363] [Q&A] [answered]
Hello there,
I am meeting trouble with the Preview of the Search Result Webpart. For now, when I put mu cursor on the preview for a Microsoft document (mainly Excel) I can only see the layout of the Online application. How can I see the document and not the action bar ?

For Word, I can scroll down and up on the preview, but for Excel, I have no view on my tables.

Thank you for helping,

Bruno
![Preview trouble](https://user-images.githubusercontent.com/89794621/131488198-902d9a76-1082-4801-b0f5-e827b0b8b247.png)

**Comment by FranckyC** [comment]
Hi @Vblgh-Bruno, with the 'Simple list' layout, you can disable the preview by leaving the **PreviewUrl** slot value empty:

![image](https://user-images.githubusercontent.com/5659122/131540830-16deea1f-9cbd-4bb5-9ade-2361b02948d8.png)

**Comment by Vblgh-Bruno** [comment]
Hello Frank, and thank you. What about if I want the preview ? it's only for Excel that the preview is not exploitable. My client wants a preview, but a preview that is readable, which is not the case for Excel sheets.

**Comment by wobba** [comment, marked as answer]
@Vblgh-Bruno One option is to create a custom template and add logic to if/else check on Excel - and how the template works per file type.
